gammapy/irf/effective_area_table.py

```diff
@@ -444,6 +446,67 @@ def read(filename):
         hdu_list = fits.open(filename)
         return EffectiveAreaTable2D.from_fits(hdu_list)
 
+    def to_effectiveareatable(self, offset, energ_lo = None, energ_hi = None):
+        """Evaluate at a given offset and return effective area table object.
+
+        The energy thresholds in the effective area table object are not set.
+        If the effective area table is intended to be used for spectral analysis,
+        the final energy binning should be given at this point, since the
+        effective area table class is not able to perform interpolation at
+        this point.
```

Comment:
I don't think its necessary to update the EffectiveAreaTable class (in this regard), if it is meant to be a wrapper around ARF files.